qa/integration/integration_tests.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'manticore'
   s.add_development_dependency 'stud'
   s.add_development_dependency 'pry'
-  s.add_development_dependency 'logstash-devutils'
+  s.add_development_dependency 'logstash-devutils', '= 1.3.3'
```

Comment:
Why the hard version pin?

Reply:
@andrewvc counter-question, why not? ... Nah seriously though, this thing pulls in a lot of dependencies and weirdly resolved to some ancient version without pinning it.
I don't see much value in letting this float freely (this code is kind of fixed and we don't expect to be updating dependencies much) => why take the risk that say `1.3.4` randomly breaks CI and we waste more time than necessary to track this down?

Reply:
+1 to lock it down to an exact version.